### DISCUSSION TITLE
Web Server and Kodi 19

### DISCUSSION THREAD

**Discussion by DeNohvo** [croneter/PlexKodiConnect#1415] [Q&A] [answered]
I updated to Kodi 19 and got an alert that Kodi has disabled the web interface. I then updated the PKC repo and got the latest addon which then once again enable the web interface, i also recall seeing a message that it will use a strong password.

When checking Kodi web interface settings, I know I change the password but how can i check what the current password is?

**Comment by croneter** [comment, marked as answer]
I don't think you can view a password already set - that would have security implications. Just set a new password. Problem solved
